semver/_shared.ts

```diff
@@ -87,30 +87,23 @@ const MAINVERSION =
 // ## Pre-release Version Identifier
 // A numeric identifier, or a non-numeric identifier.
 
-const PRERELEASEIDENTIFIER = "(?:" + NUMERICIDENTIFIER + "|" +
-  NONNUMERICIDENTIFIER + ")";
+const PRERELEASEIDENTIFIER = `(?:${NUMERICIDENTIFIER}|${NONNUMERICIDENTIFIER})`;
 
 // ## Pre-release Version
 // Hyphen, followed by one or more dot-separated pre-release version
 // identifiers.
 
-const PRERELEASE = "(?:-(" +
-  PRERELEASEIDENTIFIER +
-  "(?:\\." +
-  PRERELEASEIDENTIFIER +
-  ")*))";
+const PRERELEASE =
+  `(?:-(${PRERELEASEIDENTIFIER}(?:\\.${PRERELEASEIDENTIFIER})*))`;
 
 // ## Build Metadata Identifier
 // Any combination of digits, letters, or hyphens.
-
-const BUILDIDENTIFIER = "[0-9A-Za-z-]+";
+const BUILDIDENTIFIER = `[0-9A-Za-z-]+`;
```

Comment:
This change is unnecessary, as no string interpolation is happening here. Ditto to other instances.

Reply:
done